### PR TITLE
Fix spacing to 28px in latest TestFlight version

### PR DESCRIPTION
**Pull request by walmat**
Fixes APP-2209

## What changed (plus any additional context for devs)
Fixes spacing in areas of trending tokens and cleans up a few things in the process

## Screen recordings / screenshots
Screenshot 2024-12-18 at 5 02 09 PM

## What to test
n/a

### CODE REVIEW

File: src/components/Discover/TrendingTokens.tsx [hunk 1]
```diff
@@ -388,7 +388,7 @@ function TrendingTokenRow({ token }: { token: TrendingToken }) {
           chainSize={20}
         />
 
-        <View style={{ gap: 12, flex: 1 }}>
+        <View style={{ gap: 10, flex: 1 }}>
```
Comment (walmat) [inline src/components/Discover/TrendingTokens.tsx:391]:
```suggestion
        <View style={{ gap: 12, flex: 1 }}>
```

File: src/components/Discover/TrendingTokens.tsx [hunk 2]
```diff
@@ -590,17 +591,18 @@ function TrendingTokenData() {
 
   return (
     <FlatList
-      style={{ marginHorizontal: -20 }}
-      contentContainerStyle={{ paddingHorizontal: 20 }}
+      style={{ marginHorizontal: -20, paddingVertical: 12, marginVertical: -12 }}
```
Comment (christianbaroni) [inline src/components/Discover/TrendingTokens.tsx:594]: Is this just to expand the list's touchable area?

**Review by christianbaroni — APPROVED** [review]

**Review by derHowie — APPROVED** [review]

File: src/components/Discover/TrendingTokens.tsx (hunk 2 above)
Comment (walmat) [reply src/components/Discover/TrendingTokens.tsx:594]: yeah adds a little padding on the top/bottom of the list

**Comment by brunobar79** [comment]
Launch in [simulator](http://localhost:29070/install/ios?virtual=https://app-team.p.rainbow.me/@matthew/APP-2209/1119ac8276a4c61b1dc10220db2ecabe9d8bd9dc.app.zip) or [device](http://localhost:29070/install/ios?physical=https://app-team.p.rainbow.me/@matthew/APP-2209/1119ac8276a4c61b1dc10220db2ecabe9d8bd9dc.ipa) for 1119ac8276a4c61b1dc10220db2ecabe9d8bd9dc

**Pull request merged**